bindings/nodejs/lib/AccountManager.ts

```diff
@@ -133,6 +133,13 @@ export class AccountManager {
         return JSON.parse(response).payload;
     }
 
+    async getLedgerStatus(): Promise<NodeInfoWrapper> {
```

Comment:
Why do we return a `NodeInfoWrapper` here?

Reply:
Because it's wrong :trollface: